lib/rubocop/cop/style/ip_addresses.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require 'resolv'
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop checks for hardcoded IP addresses, which can make code
+      # brittle. IP addresses are likely to need to be changed when code
+      # is deployed to a different server or environment, which may break
+      # a deployment if forgotten. Prefer setting IP addresses in ENV or
+      # other configuration.
```

Comment:
This should have some good and bad code examples.

Reply:
Sure - how do I show a good code example for the lack of something?

Reply:
Actually I think I have an idea

Reply:
You can just use imaginary config API (e.g. some reference to a config module or env variables).

Reply:
Updated to add examples.